src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -22,13 +23,22 @@ object Lorre extends App with LazyLogging {
   lazy val db = DatabaseUtil.db
   val tezosNodeOperator = new TezosNodeOperator(TezosNodeInterface)
 
+  var iterationsOfLorre = 0
+  val feeUpdateInterval = conf.getInt("feeUpdateInterval")
+  val feeCheck = iterationsOfLorre == 0 || iterationsOfLorre % feeUpdateInterval == 0
```

Comment:
Function names should be verbs or predicates, e.g. `shouldFeeBeCalculatedThisIteration`.